notes-app-ktor/src/jvmMain/kotlin/org/avr/notes/app/v1/FolderController.kt

```diff
@@ -0,0 +1,111 @@
+package org.avr.notes.app.v1
+
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.util.*
+import kotlinx.datetime.Clock
+import org.avr.notes.api.v1.models.IFolderRequest
+import org.avr.notes.app.common.NotesAppSettings
+import org.avr.notes.common.FolderContext
+import org.avr.notes.common.helpers.asNotesError
+import org.avr.notes.common.models.NotesState
+import org.avr.notes.common.models.folder.FolderCommand
+import org.avr.notes.logging.common.INotesLoggerWrapper
+import org.avr.notes.mappers.fromRequestData
+import org.avr.notes.mappers.toTransport
+import org.avr.notes.stub.FolderStub
+
+
+class FolderController(
+    private val appSettings: NotesAppSettings,
+    private val logger: INotesLoggerWrapper
+) {
+    suspend fun createFolder(call: ApplicationCall) {
```

Comment:
Это все можно было записать в одну строчку
```suggestion
    suspend fun createFolder(call: ApplicationCall) = processFolderRequestV1(call, FolderCommand.CREATE_FOLDER)
```
А если бы не класс, то можно было обойтись без аргумента